sql-parser/src/main/java/io/crate/sql/parser/AstBuilder.java

```diff
@@ -1393,10 +1552,24 @@ private static boolean isDistinct(SqlBaseParser.SetQuantContext setQuantifier) {
         throw new IllegalArgumentException("Unsupported quantifier: " + symbol.getText());
     }
 
+    private List<QualifiedName> getIdents(List<SqlBaseParser.QnameContext> qnames) {
+        return qnames.stream().map(this::getQualifiedName).collect(toList());
+    }
+
     private static void validateFunctionName(QualifiedName functionName) {
         if (functionName.getParts().size() > 2) {
             throw new IllegalArgumentException(String.format(Locale.ENGLISH, "The function name is not correct! " +
                 "name [%s] does not conform the [[schema_name .] function_name] format.", functionName));
         }
     }
+
+    private AbstractMap.SimpleEntry<String, List<QualifiedName>> getClassAndIdentsForPrivileges(boolean onCluster,
+                                                                                                SqlBaseParser.ClazzContext clazz,
+                                                                                                List<SqlBaseParser.QnameContext> qname) {
+        if (onCluster) {
+            return new AbstractMap.SimpleEntry<>(CLUSTER, Collections.emptyList());
```

Comment:
not sure if this kind of helper method makes sense as the key/value is immediately unpacked, and using a SimpleEntry as structure seems a bit odd?

And the method could also be static, in case we keep it.